src/main/java/ladder/domain/result/ExecutionResults.java

```diff
@@ -0,0 +1,43 @@
+package ladder.domain.result;
+
+import java.util.List;
+import java.util.Objects;
+
+public class ExecutionResults {
+
+    public static final String ERROR_EMPTY_MSG = "리스트가 비어있습니다..";
+    private final List<String> results;
+
+    public ExecutionResults(List<String> results) {
+        if (results == null || results.size() == 0) {
```

Comment:
```suggestion
        if (results == null || results.isEmpty()) {
```
좀 더 가독성이 좋아질 수 있겠죠 ㅎ